### PR TITLE
Robin changes

### PR DESCRIPTION
**Pull request by viktor76525**

### CODE REVIEW

File: scripts/bb_sim.py [hunk 1]
```diff
@@ -266,14 +278,16 @@ def unfinished():
             "highpass",
         )
         lf_acc[:, c] = bwfilter(
-            ampdeamp(
```
Comment (jasonmotha) [inline scripts/bb_sim.py]: This one liner becomes hard to follow once split into mulitple lines.

Comment (shot0829) [reply scripts/bb_sim.py]: i think the typical 
```if statment:
else:
``` 
would probably be more readable
